nautobot/docs/development/jobs/job-structure.md

```diff
@@ -99,12 +99,6 @@ class ExampleJob(Job):
 
 If you code a multi-line description, the first line only will be used in the description column of the Jobs list, while the full description will be rendered in the Job detail view, submission, approval, and results pages.
 
-### `approval_required`
```

Comment:
Might be good to find somewhere in this document (either here or immediately under the `## Class Metadata Attributes` heading?) to put in a `--- 3.0.0` admonition documenting the removal of this attribute.